server/graphql-schema.js

```diff
@@ -856,6 +883,26 @@ const graphqlSchema = gql`
     }
 
     # Mutation-specific types below
+    """
+    Mutations scoped to a given Assistive Technology
+    """
+    type AtVersionOperations {
+        """
+        Create a new AT Version if one doesn't exist
+        """
+        findOrCreateAtVersion(
+            atVersion: String!
+            availability: String
+        ): AtVersion!
+        """
+        Edits the AT Version
+        """
+        editAtVersion(
+            atVersion: String!
+            availability: String
+            updateParams: AtVersionInput!
+        ): AtVersion!
+    }
```

Comment:
One thing we should figure out is how to resolve some of the contradictions between this PR and the schema proposal I made here: https://github.com/w3c/aria-at-app/pull/411. One example difference is that I had the creating and editing AT versions under the "AtOperations" type but you have those functions under an "AtVersionOperations" type.